packages/react-ui/src/index.d.ts

```diff
@@ -40,9 +40,8 @@ export {
   LegendWidgetUI,
   LEGEND_TYPES,
   NoDataAlert,
-  // undocumented components
-  LegendCategories as _LegendCategories,
-  LegendIcon as _LegendIcon,
-  LegendProportion as _LegendProportion,
-  LegendRamp as _LegendRamp
+  LegendCategories,
+  LegendIcon,
+  LegendProportion,
+  LegendRamp
```

Comment:
So @borja-munoz, are we ok with exposing these types now?

Reply:
Yes, we are documenting them in the storybook, so I think it makes sense to remove the underscore.